abn/application/application.go

```diff
@@ -0,0 +1,116 @@
+package application
+
+// application.go - type of application
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/iter8-tools/iter8/base/log"
+)
+
+// Application is an application observed in a kubernetes cluster
+type Application struct {
+	// Name is of the form namespace/name where the name is the value of the label app.kubernetes.io/name
+	Name string `json:"name" yaml:"name"`
```

Comment:
`name string`, `tracks` and `Versions: `json:"versions" yaml:"versions"`

This ensures only version data is ever serialized for the app.